lib/actions/ssh.ts

```diff
@@ -227,9 +227,9 @@ export const ssh: CommandDefinition<
 		const applicationOrDevice =
 			params.applicationOrDevice_raw || params.applicationOrDevice;
 		const bash = await import('bash');
-		// TODO: Make this typed
-		const hasbin = require('hasbin');
-		const { getSubShellCommand } = await import('../utils/helpers');
+		const { getProxyConfig, getSubShellCommand, which } = await import(
+			'../utils/helpers'
+		);
```

Comment:
The `hasbin` package was removed, and the implementation modified to use the `which` package that the CLI was already using elsewhere. The reasons were twofold:
* `hasbin` was simply not working in my Linux and macOS tests with Node 10 and Node 12, neither the master branch (before this PR) nor with/after this PR. I didn't really understand why, but I tested that `which` worked all right on Linux, macOS and Windows.
* I realised that this was the only occurrence of `hasbin` in the CLI and any dependencies, so `hasbin` could be removed from `package.json` and `npm-shrinkwrap.json`.